strongbox-resources/strongbox-common-resources/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <include resource="org/springframework/boot/logging/logback/base.xml" />
```

Comment:
@anki2189 : I think I asked this in the past and I'm not sure I got an answer to it: why are you including this `base.xml`? If anything needs to be included, it should be the `logback-spring.xml` and we should be overriding the logger levels from there.

@sbespalov : What are your thoughts?

Reply:
it’s not entirely clear to me why we need this `logback-test.xml` in general. @anki2189 can you please clarify why you think we need it?

Reply:
@sbespalov : logback-test.xml is created only to be used by test cases. I tried to include logger in `logback-spring.xml` but I am clueless why it is not working.

@carlspring : `base.xml` has to be included for importing appenders. If we include `logback-spring.xml` it will will but with errors and warnings. 
It may be related to this .https://github.com/spring-projects/spring-boot/issues/7955

Reply:
> I tried to include logger in logback-spring.xml but I am clueless why it is not working.

what you trying to include there? for now it's seems that there is no reasons for another logback configuration only for tests.

Reply:
> > I tried to include logger in logback-spring.xml but I am clueless why it is not working.
> 
> what you trying to include there? for now it's seems that there is no reasons for another logback configuration only for tests.

I want to change log level for spring test package `org.springframework.test`

Reply:
it should be possible to do this in `logback-spring.xml ` directly

Reply:
No it not working in `logback-spring.xml`

Reply:
you mean that `<logger name="org.springframework.test" level="WARN"/>` not working in `logback-spring.xml`?

Reply:
Yes, `<logger name="org.springframework.test" level="WARN"/> `not working in `logback-spring.xml`